controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -815,6 +815,23 @@ private Throwable getRealException(Throwable ex) {
                 .thenApplyAsync(res -> TxnState.newBuilder().setState(TxnState.State.valueOf(res.name())).build(), executor);
     }
 
+    /**
+     * List txnIds in completed state.
+     * @param scope scope name
+     * @param stream stream name
+     * @param limit max no of entries to get
+     * @param continuationToken continuationToken
+     * @param requestId request id
+     * @return pair having map of uuid with status and continuationToken
```

Comment:
pair having `map` or `list`?

Reply:
or it should be 
@return List of transaction response?

Reply:
Updated the doc.